pkg/apis/config/feature_flags.go

```diff
@@ -234,6 +226,26 @@ func setEnabledAPIFields(cfgMap map[string]string, defaultValue string, feature
 	return nil
 }
 
+// setEnforceNonFalsifiability sets the ""enforce-nonfalsifiability"" flag based on the content of a given map.
+// If the feature gate is invalid, then an error is returned.
+func setEnforceNonFalsifiability(cfgMap map[string]string, enableAPIFields string, feature *string) error {
+	value, err := getEnforceNonfalsifiabilityFeature(cfgMap)
```

Comment:
From reading the two functions`getEnforceNonfalsifiabilityFeature` and `setEnforceNonFalsifiability`, I feel like we might be able to merge the former to the latter.